tests/integration/api/test_search_sort_desc.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import requests
+from .constants import API_URL
+from .utils import assert_response_status
+
+@pytest.mark.parametrize("endpoint, expected_status", [
+    ("/search?query=keyword%3ANASA&sort=date%3Adesc&filter=language%3ASerbian", 200),
+])
+def test_search_language_and_date_sorting(endpoint, expected_status):
+    url = API_URL + endpoint
+    response = requests.get(url)
+
+    assert response.status_code is not None
+    assert_response_status(url, response, expected_status)
+
+    if expected_status == 200:
```

Comment:
the assert_response_status above will cause the test to fail if the expected_status is not 200. I think we can remove this if